app/models/volunteer.rb

```diff
@@ -114,17 +114,29 @@ class Volunteer < ApplicationRecord
       .merge(Assignment.inactive)
       .where.not(assignments: { volunteer_id: with_active_assignments.ids })
       .or(loj_without_assignments)
+      .merge(
+        left_outer_joins(:group_assignments)
+          .where('group_assignments.period_end IS NULL
+            OR group_assignments.period_end < ?', Time.zone.today)
```

Comment:
can't you use here the scopes where `started`, `ongoing`, `no_end` are already defined in the `group assignment` [model](https://github.com/panter/aoz-003/blob/develop/app/models/group_assignment.rb)? or was this the problem we mentioned in todays standup?

Reply:
@nuritnt no, there not compatible, because they use different SQL joins `LEFT OUTER JOIN`, `LEFT JOIN`, etc.. aren't compatible